src/index.js

```diff
@@ -15,13 +15,29 @@ const hasFullStoryWithFunction = (...testNames) => {
 };
 
 const guard = name => (...args) => {
+  if (window._fs_dev_mode) {
+    const message = 'FullStory is in dev mode and is not recording';
+    console.warn(message); // eslint-disable-line no-console
+    return message;
```

Comment:
Since this will log on every function call, would you want users to filter the logs to ignore the `message` or would it be better to have something like:

```js
const message = 'FullStory is in dev mode and is not recording';

if (window._fs_dev_mode && !window._fs_dev_msg_displayed) {
  console.warn(message); // eslint-disable-line no-console
  window._fs_dev_msg_displayed = true;
}

if (window._fs_dev_mode) {
  return message;
}
```

Reply:
Great point @rgpass. My hope is that the `console.warn`s would be infrequent enough to not pollute the console for folks working in a dev environment. On the other hand, I'm hesitant to limit the number of `console.warn`s because I want to make sure there is the maximum amount of signal available for folks who inadvertently push FullStory to prod in `devMode`. This will happen :) and the sooner folks can figure out why they aren't getting the data they expect into FullStory, the better.

Reply:
Ah, great call. I'll just filter the message then. 👍